src/ontology/cl-edit.owl

```diff
@@ -5333,13 +5333,10 @@ SubClassOf(obo:CL_0000197 obo:CL_0000003)
 # Class: obo:CL_0000198 (pain receptor cell)
 
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "MESH:D009619") obo:IAO_0000115 obo:CL_0000198 "The peripheral receptor for pain. Includes receptors which are sensitive to painful mechanical stimuli, extreme heat or cold, and chemical stimuli. All mammalian nociceptors are free nerve endings.")
-AnnotationAssertion(Annotation(rdfs:seeAlso "https://meshb.nlm.nih.gov/record/ui?ui=D009619") oboInOwl:hasExactSynonym obo:CL_0000198 "nociceptor")
-AnnotationAssertion(Annotation(rdfs:seeAlso "https://www.thefreedictionary.com/nocireceptor") oboInOwl:hasExactSynonym obo:CL_0000198 "nocireceptor")
+AnnotationAssertion(Annotation(rdfs:seeAlso "https://meshb.nlm.nih.gov/record/ui?ui=D009619"^^xsd:anyURI) oboInOwl:hasExactSynonym obo:CL_0000198 "nociceptor")
```

Comment:
should these be dbxref then? brief look at previous PR https://github.com/obophenotype/cell-ontology/pull/1558/files seems to suggest so

Reply:
Thanks for the prompt review. Logic detailed below.

Context: [Current editor guidelines](https://github.com/obophenotype/cell-ontology/blob/master/docs/editing_guidelines.md), under section titled "Technical details of adding a cross-reference using Protégé"

MESH:D009619 directs to https://id.nlm.nih.gov/mesh/D009619.html.
"MESH:D009619" would be added using database_cross_reference.

In this case, the link "https://meshb.nlm.nih.gov/record/ui?ui=D009619" is used since the page at that location more clearly justifies the synonym.

<img width="265" alt="Screenshot 2023-08-17 at 10 10 45" src="https://github.com/obophenotype/cell-ontology/assets/95643797/bd6ca67d-2888-4974-bfaa-0ff03fde2688">


Per current editor guidelines, URLs should be used sparingly, but when used are added using 'see also' with datatype xsd:anyURI.